.travis.yml

```diff
@@ -1,36 +0,0 @@
-dist: trusty
```

Comment:
How did this file manage to survive that long?

Reply:
:shrug: Sometimes I find remnants of the fork here and there.